apisix/plugins/prometheus/exporter.lua

```diff
@@ -410,7 +411,16 @@ local function shared_dict_status()
 end
 
 
+-- for test
+local function log_process_type()
```

Comment:
We can hook the `_M.collect` to avoid adding logging just for testing.

Reply:
A hook example: https://github.com/apache/apisix/blob/797bc7dbcce37eb565bb7a5207534455f90dcf82/t/cli/test_main.sh#L624

Reply:
done